routes/users.js

```diff
@@ -14,4 +15,16 @@ router.post("/candlestick", async function (req, res, next) {
   }
 });
 
+router.get("/asset/:userid", verifyToken, async function (req, res, next) {
+  const { userId } = req.params;
+
+  try {
+    const user = await User.findById({ _id: userId }).lean().exec();
+
+    res.status(200).send({ userAsset: user });
```

Comment:
저희 apiDoc 형식은 
```suggestion
    res.status(200).send({ data: user });
```
이렇게 작성이 되어 있네요 명시적으로 보이기 위하신 거라면 웅민님 의견 따르겠습니다 ~~!

Reply:
user는 유저정보 전체를 가져온것 같은데 전부 보내시는 이유가 있을까요?
userAsset이라면 user에서 asset만 꺼내서 보내주는게 자연스러울것 같습니다.

Reply:
위 설명란에 적어 놓긴 했는데 잘 전달이 안 됐나봐요
아직 저장된 것이 없어 일단 유저의 전체 데이터를 보내게 해놨습니다.
asset 데이어 처리해주는 것을 한 후 asset 데이터만 넘겨주는 작업을 하는 칸반 카드도 작성해놔 추가 작업을 통해 수정하겠습니다.